client/blocks/inline-help/popover.jsx

```diff
@@ -132,6 +155,15 @@ class InlineHelpPopover extends Component {
 					setStoredTask={ setStoredTask }
 				/>
 
+				{ 'gutenberg' === this.props.selectedEditor && (
```

Comment:
Possibly overkill nitpick, and definitely not a blocker.

If we visited `/gutenberg/post` with `selectedEditor === 'classic'`, we wouldn't see the opt out button, and at the same time if we visited `/post` with `selectedEditor === 'gutenberg'`, we would see the opt out.

A simple workaround would be to check, for example, if `state.ui.section.group === 'gutenberg'`.